<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nuevareserva_cliente</class>
 <widget class="QWidget" name="nuevareserva_cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(237, 232, 208);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);

</string>
   </property>
   <property name="text">
    <string>Numero de Comensales:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mesasDisponibles">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);
</string>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);</string>
   </property>
   <property name="text">
    <string>Hora de Reserva:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);

</string>
   </property>
   <property name="text">
    <string>Mesas Disponibles: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="numeroComensales">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);
</string>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);</string>
   </property>
   <property name="text">
    <string>Fecha de Reserva:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 24pt &quot;Georgia&quot;;
color: rgb(45, 81, 40);</string>
   </property>
   <property name="text">
    <string>NUEVA RESERVA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 81, 40);</string>
   </property>
   <property name="text">
    <string>Crear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(45, 81, 40);</string>
   </property>
   <property name="text">
    <string>Validar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
